src/ai/backend/client/func/audit_logs.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import annotations
+
+import textwrap
+from typing import Dict, Sequence, Union
+
+from ai.backend.client.output.fields import auditlog_fields
+from ai.backend.client.output.types import FieldSpec, PaginatedResult
+from ai.backend.client.pagination import generate_paginated_results
+from ai.backend.client.request import Request
+from ai.backend.client.session import api_session
+
+from .base import BaseFunction, api_function
+
+__all__ = ("AuditLog",)
+
+
+_default_list_fields = [
+    auditlog_fields["user_id"],
+    auditlog_fields["access_key"],
+    auditlog_fields["email"],
+    auditlog_fields["action"],
+    auditlog_fields["data"],
+    auditlog_fields["target_type"],
+    auditlog_fields["target"],
+    auditlog_fields["created_at"],
+]
+
+
+class AuditLog(BaseFunction):
+    """
+    Provides management of audit logs.
+    """
+
+    @api_function
+    @classmethod
+    async def list(cls, list_all=False):
+        rqst = Request("GET", "/audit_logs")
```

Comment:
Hmm, where's handler definition for this API located at? I only can find graphql implementations.